client/src/components/icons/MainLogo.svelte

```diff
@@ -2,8 +2,15 @@
     import './../../components/icons/icon-styles.css';
 
     export let alt: string;
+    export let mini = false as boolean;
 
     const iconUrl = new URL('../../assets/icons/doctrack.svg', import.meta.url);
 </script>
 
-<img {alt} src={iconUrl.pathname} on:click on:keydown />
+<img {alt} class:mini={mini} src={iconUrl.pathname} on:click on:keydown />

```

Comment:
If I'm not mistaken, Svelte allows this shorthand.
```suggestion
<img {alt} class:mini src={iconUrl.pathname} on:click on:keydown />
```